### PR TITLE
KubernetesConfiguration 2022-04-02-preview version changes

### PR DESCRIPTION
**Pull request by RamyasreeChakka**
### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify

  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
         We plan to deploy the changes in March 2022.

  4. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
         We target swagger publish in March 2022.

  6. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [x] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/kubernetesconfiguration/resource-manager/readme.md [hunk 1]
```diff
@@ -28,11 +28,24 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2022-03
+tag: package-preview-2022-04
 ```
```
Comment (RamyasreeChakka) [inline specification/kubernetesconfiguration/resource-manager/readme.md:32]: Lets use package-2022-03 as the default api version

File: specification/kubernetesconfiguration/resource-manager/readme.md [hunk 2]
```diff
@@ -59,19 +72,6 @@ input-file:
   - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/operations.json
 ```
 
-### Tag: package-preview-2021-11
-
```
Comment (RamyasreeChakka) [inline specification/kubernetesconfiguration/resource-manager/readme.md:63]: Can you please revert this change?

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 3]
```diff
@@ -0,0 +1,1095 @@
...
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.KubernetesConfiguration/privateLinkScopes": {
+      "get": {
+        "description": "Gets a list of all Azure Arc PrivateLinkScopes within a subscription.",
+        "operationId": "PrivateLinkScopes_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:46]: >v1/types.json [](http://example.com/codeflow?start=69&length=13)

Any reason not to use v3/types.json?

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 4]
```diff
@@ -0,0 +1,1095 @@
...
+          "200": {
+            "description": "A list containing 0 or more Azure Arc PrivateLinkScope definitions.",
+            "schema": {
+              "$ref": "#/definitions/KubernetesConfigurationPrivateLinkScopeListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:62]: >v2/types.json [](http://example.com/codeflow?start=71&length=13)

Consider v3/types.json

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 5]
```diff
@@ -0,0 +1,1095 @@
...
+        "x-ms-examples": {
+          "PrivateLinkScopeGet": {
+            "$ref": "./examples/PrivateLinkScopesGetValidationForMachine.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PrivateLinkScopesResource": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:670]: >PrivateLinkScopesResource [](http://example.com/codeflow?start=5&length=25)

Recommend using TrackedResource definition from common-types.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 6]
```diff
@@ -0,0 +1,1095 @@
...
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/PrivateLinkScopesResource"
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:738]: >PrivateLinkScopesResource [](http://example.com/codeflow?start=33&length=25)

If you use the TrackedResource definition from common-types v3/types.json, it already includes systemData.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 7]
```diff
@@ -0,0 +1,1095 @@
...
+        }
+      ],
+      "description": "An Azure Arc PrivateLinkScope definition."
+    },
+    "PrivateLinkScopeValidationDetails": {
+      "type": "object",
+      "properties": {
+        "id": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:746]: >id [](http://example.com/codeflow?start=9&length=2)

What does this id identify?

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 8]
```diff
@@ -0,0 +1,1095 @@
...
+          "items": {
+            "$ref": "#/definitions/ConnectionDetail"
+          }
+        }
+      }
+    },
+    "ConnectionDetail": {
+      "type": "object",
+      "properties": {
+        "id": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:767]: >id [](http://example.com/codeflow?start=9&length=2)

I think this id is of the linked resource, not this connection detail object. Could you change it to connectedResourceId or similar? Id implies it's identifying the ConnectionDetail object it belongs to.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 9]
```diff
@@ -0,0 +1,1095 @@
...
+    "KubernetesConfigurationPrivateLinkScopeProperties": {
+      "type": "object",
+      "description": "Properties that define a Azure Arc PrivateLinkScope resource.",
+      "required":["clusterArmId"],
+      "properties": {
+        "publicNetworkAccess": {
+          "description": "Indicates whether machines associated with the private link scope can also use public Azure Arc service endpoints.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "provisioningState": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:805]: >provisioningState [](http://example.com/codeflow?start=9&length=17)

This should be a string enum.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 10]
```diff
@@ -0,0 +1,1095 @@
...
+          "type": "string",
+          "description": "Current state of this PrivateLinkScope: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Provisioning ,Succeeded, Canceled and Failed.",
+          "readOnly": true
+        },
+        "clusterArmId": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:810]: >clusterArmId [](http://example.com/codeflow?start=9&length=12)

Don't include abbreviations in property names. Consider using clusterResourceId for this property.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 11]
```diff
@@ -0,0 +1,1095 @@
...
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:884]: >v1/types.json [](http://example.com/codeflow?start=67&length=13)

If you use Resource definition from common-types v3/types.json it already includes systemData.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 12]
```diff
@@ -0,0 +1,1095 @@
...
+        }
+      ]
+    },
+    "PrivateLinkResourceProperties": {
+      "type": "object",
+      "description": "Properties of a private link resource.",
+      "properties": {
+        "groupId": {
+          "description": "The private link resource group id.",
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:893]: >The private link resource group id [](http://example.com/codeflow?start=26&length=34)

Would be good to describe the nature of this identifier (e.g. whether it's an ARM resourceId or guid, or...)? If it's an ARM resourceId, then groupResourceId would be a better name.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 13]
```diff
@@ -0,0 +1,1095 @@
...
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointConnectionDataModel": {
+      "type": "object",
+      "description": "The Data Model for a Private Endpoint Connection associated with a Private Link Scope",
+      "properties": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:918]: >properties [](http://example.com/codeflow?start=7&length=10)

Consider using Resource from common-types

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 14]
```diff
@@ -0,0 +1,1095 @@
...
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointProperty": {
+      "type": "object",
+      "description": "Private endpoint which the connection belongs to.",
+      "properties": {
+        "id": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:963]: >id [](http://example.com/codeflow?start=9&length=2)

privateEndpointResourceId?

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 15]
```diff
@@ -0,0 +1,1095 @@
...
+    },
+    "PrivateLinkServiceConnectionStateProperty": {
+      "type": "object",
+      "description": "State of the private endpoint connection.",
+      "required": [
+        "status",
+        "description"
+      ],
+      "properties": {
+        "status": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:977]: >status [](http://example.com/codeflow?start=9&length=6)

Does this have a fixed set of values? If so, it should be a string enum.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 16]
```diff
@@ -0,0 +1,1095 @@
...
+          "description": "The private link service connection status.",
+          "type": "string"
+        },
+        "description": {
+          "description": "The private link service connection description.",
+          "type": "string"
+        },
+        "actionsRequired": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:985]: >actionsRequired [](http://example.com/codeflow?start=9&length=15)

Is this a collection? If so, it's better to use an array rather than encoding the actions in a special string format.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 17]
```diff
@@ -0,0 +1,1095 @@
...
+          "readOnly": true
+        }
+      }
+    },
+    "PrivateEndpointConnection": {
+      "type": "object",
+      "description": "A private endpoint connection",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:997]: >/v1/types.json [](http://example.com/codeflow?start=66&length=14)

This and the systemData property can be replaced with Resource from common-types v3/types.json.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 18]
```diff
@@ -0,0 +1,1095 @@
...
+    "PrivateLinkScopeId": {
+      "name": "privateLinkScopeId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The id (Guid) of the Azure Arc PrivateLinkScope resource.",
+      "x-ms-parameter-location": "method"
+    },
+    "GroupNameParameter": {
+      "name": "groupName",
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:1073]: >groupName [](http://example.com/codeflow?start=15&length=9)

Why not privateLinkName? What kind of group is it?

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json (hunk 14 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:963]: Updated.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json (hunk 18 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:1073]: This is defined in NRP contract: they have to have group per connections
[https://microsoft-my.sharepoint.com/:w:/p/sakulal/Eb17pOEskn1JoKpQM7dpm7sBcxWvnYAoTekqcGf_gUm3og?e=Vvgru8](https://microsoft-my.sharepoint.com/:w:/p/sakulal/Eb17pOEskn1JoKpQM7dpm7sBcxWvnYAoTekqcGf_gUm3og?e=Vvgru8) section#6.3.2

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 19]
```diff
@@ -0,0 +1,701 @@
...
+      "properties": {
+        "publicNetworkAccess": {
+          "description": "Indicates whether machines associated with the private link scope can also use public Azure Arc service endpoints.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "provisioningState": {
+          "description": "Current state of this PrivateLinkScope: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Provisioning ,Succeeded, Canceled and Failed.",
+          "type": "string",
+          "readOnly": true,
+          "$ref": "../../common/2022-03-01/definitions.json#/definitions/ProvisioningStateDefinition"
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:618]: >2022-03-01 [](http://example.com/codeflow?start=32&length=10)

Recommend adding definitions.json to new API version folder even if it hasn't changed.

**Comment by mentat9** [comment]
@kulalking - Signing off with comments from ARM perspective. Just one open comment recommending adding definitions.json to the new API version folder rather than referring back to previous API version.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 20]
```diff
@@ -0,0 +1,701 @@
...
+        },
+        "x-ms-examples": {
+          "PrivateLinkScopeUpdateTagsOnly": {
+            "$ref": "./examples/PrivateLinkScopesUpdateTagsOnly.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KubernetesConfiguration/privateLinkScopes/{scopeName}/privateLinkResources": {
+      "get": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:297]: >get [](http://example.com/codeflow?start=7&length=3)

Recommend keeping x-ms-pageable. Even if your API doesn't support paging yet, if you use x-ms-pageable with null for nextLinkName, you can add paging later without needing a new API version.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json [hunk 21]
```diff
@@ -0,0 +1,701 @@
...
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Deletes a private endpoint connection with a given name.": {
+            "$ref": "./examples/PrivateEndpointConnectionDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KubernetesConfiguration/privateLinkScopes/{scopeName}/privateEndpointConnections": {
+      "get": {
```
Comment (mentat9) [inline specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:532]: >get [](http://example.com/codeflow?start=7&length=3)

Recommend keeping x-ms-pageable

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json (hunk 19 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:618]: Can you please explain more how does this help.

File: specification/kubernetesconfiguration/resource-manager/readme.md (hunk 1 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/readme.md]: done.

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json (hunk 20 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json:297]: Fixed

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json (hunk 8 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json]: using common so got fixed

File: specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json (hunk 7 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json]: using common got fixed

File: specification/kubernetesconfiguration/resource-manager/readme.md (hunk 2 above)
Comment (kulalking) [reply specification/kubernetesconfiguration/resource-manager/readme.md]: done

**Comment by kulalking** [comment]
Thanks a lot or the review , learned a lot.

---
In reply to: [1067305969](https://github.com/Azure/azure-rest-api-specs/pull/18138#issuecomment-1067305969) [](http://example.com/codeflow?ancestors=1067305969,1067305969)

**Comment by kulalking** [comment]
Fixed.

---
In reply to: [1062426617](https://github.com/Azure/azure-rest-api-specs/pull/18138#issuecomment-1062426617) [](http://example.com/codeflow?ancestors=1062426617,1062426617)

**Review by ruowan — APPROVED** [review]
